lib/CleantalkSP/SpbctWP/ListTable.php

```diff
@@ -168,20 +168,26 @@ public function getData()
             foreach ( $this->columns_names as $columns_name ) {
                 $columns[] = $this->sql['table'] . '.' . $columns_name;
             }
+            $limit_value = '';
+            $limit_sql = '';
+            if ($this->sql['limit'] != false) {
```

Comment:
Please, use the strict comparison. You have strictly set false on the code below, isn't it?

https://github.com/CleanTalk/security-malware-firewall/pull/318/files#diff-404405e00e28d72e5b4d80d3d8509b0f94477f01eed1e23885caacc474910275R856